pkg/nodelabels/builder.go

```diff
@@ -84,5 +84,6 @@ func BuildMandatoryControlPlaneLabels() map[string]string {
 	nodeLabels[RoleLabelControlPlane20] = ""
 	nodeLabels[RoleLabelName15] = RoleMasterLabelValue15
 	nodeLabels["kops.k8s.io/kops-controller-pki"] = ""
+	nodeLabels["node.kubernetes.io/exclude-from-external-load-balancers"] = ""
```

Comment:
Should we conditionally add this based on kubernetes minor version?

Reply:
If people have enabled the new service logic, they need this label as well. If they haven't, this is no-op. So my preference is to do this unconditionally.

Reply:
We have the logic when Masters hosts ingresses, can we somehow disable this behaviour?